src/MQTTAsyncUtils.c

```diff
@@ -2037,6 +2037,32 @@ static int MQTTAsync_completeConnection(MQTTAsyncs* m, Connack* connack)
 	return rc;
 }
 
+static void MQTTAsync_freePack(MQTTPacket* pack)
```

Comment:
Depending on type that pack represents, other data members may need to be freed. Call the appropriate functions to do so.